app/views/posts/show.html.erb

```diff
@@ -43,11 +43,12 @@
       <%= tabs_content_tag("description", "", "") do %>
         <%= render "carousel" if @ordered_images.present? || @post.carousel_video.present? %>
 
-        <% if @post.description.present? || @post.source_derivs.any? %>
+        <% if @post.description.present? || @post.source_derivs.any? || @archive_authorization.present? %>
           <div class="item__content item__description">
             <%= sanitized_markdown(@post.description) %>
 
             <%= render "derivatives/derivatives" if @post.source_derivs.any? %>
+            <%= render "archive", post: @post if @archive_authorization.present? %>
```

Comment:
Please double check if this works fine with caching, it might cache the unauthorized version no matter what

Reply:
Turns out your suspicions were correct. If an authorization is added after the post is cached, the partial is not rendered.

Reply:
This issue is actually not relevant because transferring the post will update the post's `updated_at` timestamp.